cocos/2d/CCNode.cpp

```diff
@@ -1299,8 +1240,14 @@ void Node::visit()
     visit(renderer, parentTransform, true);
 }
 
-uint32_t Node::processParentFlags(const Mat4& parentTransform, uint32_t parentFlags)
+uint32_t Node::processParentFlags(const Mat4& parentTransform, uint32_t parentFlags, bool updateTransform)
```

Comment:
`updateTransform` is nothing about parent's flags. It just uses existing mechanism to update Node's transform caused by physics engine. It is strange.